src/mock-api-test/java/com.smartsheet.api/sdk_test/SheetTests.java

```diff
@@ -0,0 +1,49 @@
+package com.smartsheet.api.sdk_test;
+/*
+ * #[license]
+ * Smartsheet SDK for Java
+ * %%
+ * Copyright (C) 2014 Smartsheet
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * %[license]
+ */
+
+
+import com.smartsheet.api.Smartsheet;
+import com.smartsheet.api.models.*;
+
+import org.junit.Assert;
+import org.junit.Test;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class SheetTests {
+
+	
+	@Test
+	public void ListSheets_NoParams() throws Exception
+	{
+		Smartsheet ss =  HelperFunctions.SetupClient("List Sheets - No Params");
+		PagedResult<Sheet> sheets = ss.sheetResources().listSheets(null, null, null);
+		boolean foundObject = false;
+		for(Sheet sheet: sheets.getData()){
```

Comment:
Is there a reason you don't just check `sheets.getData()[0].getName().equals('...')`?

Reply:
I am usually of the mindset that array's in JSON does not keep the order, but it does. Only the parameter names do not keep order. I will update to make it the easier to write code.

Reply:
Okay, feel free to merge once you finish